index.ts

```diff
@@ -0,0 +1,60 @@
+import express from "express";
+import bodyParser from "body-parser";
+import {
+  deleteUser,
+  getUsers,
+  postUser,
+  putUser,
+} from "./controller/controller";
+import { port } from "./infrastructureVariables";
+const app = express();
+
+app.use(bodyParser.urlencoded());
+
+app.get("/", async (req, res) => {
+  try {
+    const users = await getUsers();
+    res.type("json").send(users);
+  } catch (err) {
+    res.status(500).send(err);
+  }
+});
+
+app.post("/post", async (req, res) => {
+  try {
+    const result: string = await postUser(
+      req.body.id,
+      req.body.name,
+      req.body.age
+    );
+    res.type("plain").status(201).send(result);
+  } catch (err) {
+    res.status(500).send(err);
+  }
+});
+
+app.put("/put", async (req, res) => {
+  try {
+    const result: string = await putUser(
+      req.body.id,
+      req.body.name,
+      req.body.age
+    );
+    res.type("plain").status(201).send(result);
```

Comment:
Is 201 status for put request?
https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete

Reply:
Changed